### DISCUSSION TITLE
Reset View Button & File Paths for both files in split view

### DISCUSSION THREAD

**Discussion by dev-rish** [moshfeu/vscode-compare-folders#87] [Q&A]
Is there any button that can simply reset the view(basically removing the 2 selected folders from comparision window) if not it would be great to have a button along side refresh and other buttons. Also when i am in split view is there a way to show breadcrums (filepath) for both files opened in the editor(left and right).  Also a cross button after every file/folder button to remove that from say the differences window would be great just like the vs code's search view

**Comment by moshfeu** [comment]
Thank you for the feedback!

1. There is not. If I understand correctly, the flow you're suggesting is 1. Compare 2 folders. 2. Reset. 3. Compare another 2 folders. If so, in order to compare another 2 folders you don't have to reset, you can just pick another pair of folders.
2. The extension uses the built-in diff view of vscode. I'm not sure it's possible to have 2 breadcrumbs but I'll check.
3. This is a great idea. Actually it was the first issue I created but I was lazy to implement and waited for a request from the crowd, now I have a reason 🙃

**Reply by dev-rish** [reply]
Firstly thank you for the great extension and quick reply really appreciate it 😀😀

1. Yes, you understood correctly. Although i did realize that but I just feel it might give more clearity (or peace of mind 😅) while using the extension. Also I noticed that there is no loading indicator when you select 2 folders for click "Compare" unlike the refresh button which shows loader in bottom right (correct me if I am wrong). Also since I didnt put _node_modules_ in _excludeFilter_, it seemed as if things remain stale if folder has many files and folders. It was only after trying 7-8 times that I finally saw things working which might be because of me putting _node_modules_ in _excludeFilter_ made folders much ligher. Which might be just because while loading contents of 2 folders initially, it doesn't show loader.
2. I mentioned this for 1 reason. Not being able to see paths of both files together. But for most part setting _diffViewTitle_ to _full path_ solved this issue. Although I did some searching and you are pretty much right about 2 Breadcrums not being visible simultaneously. But I wonder if it would be possible to show difference in the same way like when you select _Split Right_ from context menu of file tab. That's the only view I found which shows Breadcrums for both files. (Some option in settings would be great if possible).
3. Yup...that could be handy. I atleast use it often to trim the search results.

I am really looking forward for this extension to improve, test and give constructive feedback and hope you get time to work on some of these 😁 🥂

**Reply by moshfeu** [reply]
Sorry for leaving the conversation, I guess it dropped from my mind 😓 .

1. Please create an issue for the reset button. About the loader, I'm happy to share that other people asked it (#95) and it's going to be in the next version. Another new feature in the next release you might like is respecting .gitignore in order to exclude files (#96)
2. Unfortunately, diff view can't work with splitting the windows :\
3. I re-opened the issue that we were talking about (#2). I hope to get to it soon.
